src/TreeMap/App.svelte

```diff
@@ -1,593 +1,615 @@
-                            tree.setAttributeNS(null, "height", 12.598);
-                            tree.setAttributeNS(
-                                null,
-                                "viewBox",
-                                "0 0 10.5 12.598"
-                            );
-
-                            var pathNS = "http://www.w3.org/2000/svg";
-                            var path = document.createElementNS(pathNS, "path");
-                            path.setAttributeNS(
-                                null,
-                                "d",
-                                "M9,15.1V12.235a2.9,2.9,0,0,1-1,.175,3.033,3.033,0,0,1-2.093-5.2,3.055,3.055,0,0,1-.24-1.19,3,3,0,0,1,5.5-1.68h.167a3.7,3.7,0,1,1-1,7.252v3.5Z"
-                            );
-                            path.setAttributeNS(
-                                null,
-                                "transform",
-                                "translate(-4.75 -2.75)"
-                            );
-                            path.setAttributeNS(null, "fill", primaryColor);
-                            path.setAttributeNS(null, "stroke", "#fff");
-                            path.setAttributeNS(null, "stroke-width", 0.5);
-                            tree.appendChild(path);
-                            el.appendChild(tree);
-
-                            el.className = "marker";
-
-                            // make a marker for each feature and add to the map
-                            new mapboxgl.Marker(el, {
-                                anchor: "bottom",
-                            })
-                                .setLngLat(contribution.geometry.coordinates)
-                                .addTo(map);
-                        }
-                    });
-                }
-            });
+        map.addLayer({
+          id: "contrib",
+          type: "circle",
+          source: "contributions",
+          filter: ["!", ["has", "point_count"]],
+          paint: {
+            "circle-color": counterBGColor,
+            "circle-radius": [
+              "step",
+              ["to-number", ["get", "treeCount"]],
+              15,
+              50,
+              20,
+              100,
+              30,
+            ],
+            "circle-stroke-width": 4,
+            "circle-stroke-color": "#fff",
+          },
         });
-    };
+        map.addLayer({
+          id: "contrib-label",
+          type: "symbol",
+          source: "contributions",
+          filter: ["!", ["has", "point_count"]],
+          layout: {
+            "text-field": [
+              "number-format",
+              ["to-number", ["get", "treeCount"]],
+              { "max-fraction-digits": 1 },
+            ],
+            "text-font": ["Ubuntu Bold"],
+            "text-size": 12,
+          },
+          paint: {
+            "text-color": "#fff",
+          },
+        });
+        // map.loadImage("/images/tree.png", function (error, image) {
```

Comment:
Do you plan to use this code before merging this PR?

Reply:
No removing